instrumentation/vertx-reactive-3.5/javaagent/build.gradle.kts

```diff
@@ -10,12 +11,21 @@ muzzle {
   }
 }
 
-//The first Vert.x version that uses rx-java 2
-val vertxVersion = "3.5.0"
+testSets {
+  create("version35Test")
```

Comment:
Do we need a separate test set for this? Isn't it just "normal" test?

Reply:
I'm afraid we do. Due to changes in api tests for 3.5 don't build with 4.x libraries so we need to make sure we use tests only with matching library version.